lib/src/migrators/module.dart

```diff
@@ -310,24 +309,24 @@ class _ModuleMigrationVisitor extends MigrationVisitor {
 
     // If entrypoint exposes no members, it should still be forwarded to ensure
     // that the import-only file still includes its CSS.
-    var dependency =
-        _absoluteUrlToDependency(entrypoint, relativeTo: importOnlyUrl).item1;
+    var (dependency, _) =
+        _absoluteUrlToDependency(entrypoint, relativeTo: importOnlyUrl);
     var forwards = forwardsByUrl.remove(entrypoint);
     var entrypointForwards = forwards != null
         ? _forwardRulesForShown(entrypoint, '"$dependency"', forwards, {})
         : ['@forward "$dependency"'];
     var tuples = [
```

Comment:
Why have a separate variable for this rather than looping over `forwardsByUrl` directly in the `forwardLines` initializer?

Reply:
Mostly just an artifact of incrementally moving from however this was originally written -> collection literals -> patterns.

I've updated this to combine them. Is using a single-option switch expression to extract `ruleUrl` a good idea here, or should I just inline the `_absoluteUrlToDependency` call?

Reply:
I'd just inline the call. I usually only resort to fake-`let` patterns if I need to re-use the variable (also, I'd use `if`/`case` over `switch` for that).